### PR TITLE
Queue ai animation

### PR DESCRIPTION
**Pull request by ivantjh**

### CODE REVIEW

**Comment by ivantjh** [comment]
Somebody merge this if no problem

File: src/gameControl.h [hunk 1]
```diff
@@ -24,8 +26,10 @@ class GameControl {
 	int turnsElapsed;
 	bool enemyAnimating;
 	std::vector<Entity*> monsterVec;
+	std::queue<Entity*> animationQueue;
```
Comment (chesterhow) [inline src/gameControl.h:30]: This is more of an `enemyQueue` rather than an `animationQueue` isn't it?

Comment (chesterhow) [reply src/gameControl.h:30]: Up to you to decide. The rest is fine, you can merge this.

Comment (ivantjh) [reply src/gameControl.h:30]: animationQueue, it empties after animation is done

Comment (chesterhow) [reply src/gameControl.h:30]: but the queue contains the list of enemies doesnt it

File: src/level.cpp [hunk 2]
```diff
@@ -1,5 +1,9 @@
 #include "level.h"
 
+bool operator==(const Coordinates& lhs, const Coordinates& rhs) {
```
Comment (chesterhow) [inline src/level.cpp:3]: where is this used

Comment (ivantjh) [reply src/level.cpp:3]: check if guy reaches victory tile

**Pull request merged**